src/unix/linux_like/linux/mod.rs

```diff
@@ -3638,6 +3733,58 @@ pub const HWTSTAMP_FILTER_PTP_V2_SYNC: ::c_uint = 13;
 pub const HWTSTAMP_FILTER_PTP_V2_DELAY_REQ: ::c_uint = 14;
 pub const HWTSTAMP_FILTER_NTP_ALL: ::c_uint = 15;
 
+// linux/ptp_clock.h
+pub const PTP_MAX_SAMPLES: ::c_uint = 25;
+
+cfg_if! {
+    if #[cfg(any(target_arch = "powerpc", target_arch = "powerpc64"))] {
+        pub const PTP_CLOCK_GETCAPS: ::c_uint = 0x40503d01;
+        pub const PTP_EXTTS_REQUEST: ::c_uint = 0x80103d02;
+        pub const PTP_PEROUT_REQUEST: ::c_uint = 0x80383d03;
+        pub const PTP_ENABLE_PPS: ::c_uint = 0x80043d04;
+        pub const PTP_SYS_OFFSET: ::c_uint = 0x83403d05;
+        pub const PTP_PIN_GETFUNC: ::c_uint = 0xc0603d06;
+        pub const PTP_PIN_SETFUNC: ::c_uint = 0x80603d07;
+        pub const PTP_SYS_OFFSET_PRECISE: ::c_uint = 0xc0403d08;
```

Comment:
these are defined with some macros. There are rust versions here https://github.com/torvalds/linux/blob/8a696a29c6905594e4abf78eaafcb62165ac61f1/rust/kernel/ioctl.rs#L13

should those be used instead?

Reply:
I think duplicating the macros from `ioctl.h` would be better here - these numbers are kind of difficult to verify by hand as-is.